sympy/series/fourier.py

```diff
@@ -440,16 +438,213 @@ def __sub__(self, other):
         return self.__add__(-other)
 
 
-class FiniteFourierSeries(Basic):
-    def __new__(cls, *args):
-        obj = Basic.__new__(cls, *args)
-        return obj
+class FiniteFourierSeries(FourierSeries):
+    r"""Represents Finite Fourier sine/cosine series.
 
-    def truncate(self, n=3):
-        return Add(*self._args)
+       For how to compute Fourier series, see the :func:`fourier_series`
+       docstring.
+
+       Parameters
+       ==========
+       f : Expr
+            Expression for finding fourier_series
+
+       limits : ( x, start, stop)
+            x is the independent variable for the expression f
+            (start, stop) is the period of the fourier series
+
+       exprs: (a0, an, bn) or Expr
+            a0 is the constant term a0 of the fourier series
+            an is a dictionary of coefficients of cos terms
+             `an[k] = coefficient of cos(pi*(k/L)*x)`
```

Comment:
There could be double backquotes as LaTeX is not needed.

Reply:
I will update.